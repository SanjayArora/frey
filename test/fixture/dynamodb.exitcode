--> Will run: [ 'plan->findTomlFiles', 'plan->readTomlFiles', 'plan->mergeToml', 'plan->splitToml', 'plan->gatherTerraformArgs', 'plan->run' ]
--> mbp - plan->findTomlFiles
--> mbp - plan->readTomlFiles
--> mbp - plan->mergeToml
--> mbp - plan->splitToml
