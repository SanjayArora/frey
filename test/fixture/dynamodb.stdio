--> Will run: [ 'runtime', 'prepare', 'refresh', 'validate', 'plan' ]
--> {hostname} - runtime
--> {hostname} - prepare
Creating private key '{tmpdir}/frey-dynamodb.pem'
Creating public key '{tmpdir}/frey-dynamodb.pub'
Found 'terraform' with version '0.6.3'
Found 'terraformInventory' with version '0.5.0'
Found 'pip' with version '7.1.2'
Found 'ansible' with version '1.9.2'
--> {hostname} - refresh
--> {hostname} - validate
--> {hostname} - plan
The Terraform execution plan has been generated and is shown below.
Resources are shown in alphabetical order for quick scanning. Green resources
will be created (or destroyed and then created if an existing resource
exists), yellow resources are being changed in-place, and red resources
will be destroyed.

Your plan was also saved to the path below. Call the "apply" subcommand
with this plan file and Terraform will exactly execute this execution
plan.

Path: {root}/test/scenario/dynamodb/.frey/residu/terraform.plan

+ aws_dynamodb_table.basic-dynamodb-table
    arn:                                                   "" => "<computed>"
    attribute.#:                                           "" => "1"
    attribute.301566416.name:                              "" => "TopScore"
    attribute.301566416.type:                              "" => "N"
    global_secondary_index.#:                              "" => "1"
    global_secondary_index.505842399.hash_key:             "" => "GameTitle"
    global_secondary_index.505842399.name:                 "" => "GameTitleIndex"
    global_secondary_index.505842399.non_key_attributes.#: "" => "1"
    global_secondary_index.505842399.non_key_attributes.0: "" => "UserId"
    global_secondary_index.505842399.projection_type:      "" => "INCLUDE"
    global_secondary_index.505842399.range_key:            "" => "TopScore"
    global_secondary_index.505842399.read_capacity:        "" => "10"
    global_secondary_index.505842399.write_capacity:       "" => "10"
    hash_key:                                              "" => "UserId"
    name:                                                  "" => "GameScores"
    range_key:                                             "" => "GameTitle"
    read_capacity:                                         "" => "20"
    write_capacity:                                        "" => "20"
Plan: 1 to add, 0 to change, 0 to destroy.
--> Saved plan as '{root}/test/scenario/dynamodb/.frey/residu/terraform.plan'
Done
