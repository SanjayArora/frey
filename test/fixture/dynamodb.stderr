
--> Exiting with error: Error while checking for '/Users/kvz/code/frey/test/scenario/dynamodb/.git'
--> Details:

undefined
